src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -73,6 +391,42 @@ def __init__(self, sc, *args):
             self.sqlCtx = SQLContext(sc)
         except Py4JError:
             traceback.print_exc()
+            
+    def start(self):
```

Comment:
Can you define all of the instance variables here in the `__init__` function above with initial defaults?